### DISCUSSION TITLE
The Linux Foundation mentorship

### DISCUSSION THREAD

**Discussion by AdityaRaimec22** [jaegertracing/jaeger#4638] [General]
Hello folks, I want to know whether I can submit my proposal now also or the deadline has passed for linux foundation mentorship program, I want to be a mentee in that. And if yes then where to submit the proposal. Thanks

**Comment by yurishkuro** [comment]
the deadline is Aug 15

**Reply by yurishkuro** [reply]
https://github.com/jaegertracing/jaeger/issues/4457
